third_party/move/move-model/src/builder/module_builder.rs

```diff
@@ -1278,9 +1278,8 @@ impl<'env, 'translator> ModuleBuilder<'env, 'translator> {
                     let field_sym = et.symbol_pool().make(field_name_);
                     let field_ty = et.translate_type(ty);
                     let field_ty_loc = et.to_loc(&ty.loc);
+                    // store the `field_ty` and f`ield_ty_loc `to process with `ability_check_field` outside of this loop, to avoid borrow issues
```

Comment:
This comment seems to have misplaced tick marks.

Reply:
Also, this comment goes to column 146, which is too wide. Please break up.

Reply:
done